### ISSUE TITLE
Getting zero Objects in visibleCells and visibleCellIndices 

### ISSUE THREAD

**Issue by Quamber** [AlanQuatermain/AQGridView#202]
i am working on a a project where i had to add gridView with a Xib file and on selection of a cell i move to next gridView with a Xiv. the problem is that even after the data has been updated ; view has been loaded; the gridView returns zero Objects in visibleCells and visibleCellIndices. I have added reloadData Completion delegare and called the functions after that but still of no use. Please help me out. I will add the code where ever you guy's think there could be a problem.

**Comment by evadne** [comment]
$0.01 says there is no data source.
- *

On Jun 28, 2013, at 4:53, Quamber notifications@github.com wrote:

i am working on a a project where i had to add gridView with a Xib file and
on selection of a cell i move to next gridView with a Xiv. the problem is
that even after the data has been updated ; view has been loaded; the
gridView returns zero Objects in visibleCells and visibleCellIndices. I
have added reloadData Completion delegare and called the functions after
that but still of no use. Please help me out. I will add the code where
ever you guy's think there could be a problem.

—
Reply to this email directly or view it on
GitHubhttps://github.com/AlanQuatermain/AQGridView/issues/202
.

**Comment by Quamber** [comment]
well all the cells are loaded properly. right now i have set total cells to 10 and they are displayed. But whenever i try to get those it does not return any object. I was working on an animation update to this library but i have checked that source too. there is no problem in that

**Comment by evadne** [comment]
Problem contained within your own app and the sample app is fine. Correct?
- *

On Jun 28, 2013, at 5:02, Quamber notifications@github.com wrote:

well all the cells are loaded properly. right now i have set total cells to
10 and they are displayed. But whenever i try to get those it does not
return any object. I was working on an animation update to this library but
i have checked that source too. there is no problem in that

—
Reply to this email directly or view it on
GitHubhttps://github.com/AlanQuatermain/AQGridView/issues/202#issuecomment-20184282
.

**Comment by Quamber** [comment]
Yes. But the example app is not a Xib based Project. And i am trying to animate the Expansion Demo Function where i need the visible Cells.

**Comment by evadne** [comment]
So cells are displayed but the method call does not return any of them. I
suggest hooking up the debugger and watching for ivar changes to be sure
about the data source not being changed.
- *

On Jun 28, 2013, at 5:05, Quamber notifications@github.com wrote:

Yes. But the example app is not a Xib based Project. And i am trying to
animate the Expansion Demo Function where i need the visible Cells.

—
Reply to this email directly or view it on
GitHubhttps://github.com/AlanQuatermain/AQGridView/issues/202#issuecomment-20184368
.

**Comment by Quamber** [comment]
okay Datasource is set no where except the Xib files as i have set my project according to your Video on Vimeo

**Comment by evadne** [comment]
Inspect with debugger.
- *

On Jun 28, 2013, at 5:08, Quamber notifications@github.com wrote:

okay Datasource is set no where except the Xib files as i have set my
project according to your Video on Vimeo

—
Reply to this email directly or view it on
GitHubhttps://github.com/AlanQuatermain/AQGridView/issues/202#issuecomment-20184506
.

**Comment by Quamber** [comment]
okay will let you know after complete inspection. Thanks for ur Valuable Time :)
